tests/func/test_api.py

```diff
@@ -26,7 +25,7 @@ def test_get_url(tmp_dir, dvc, remote):
 
 
 @pytest.mark.parametrize("cloud", clouds)
-def test_get_url_external(erepo_dir, cloud):
+def test_get_url_external(tmp_dir, erepo_dir, cloud):
```

Comment:
In this case, api.get_url() has been returning non-existing path - cache was on `C` drive, while SSH server was on `D` (in case of github actions + Win). I think that we should check whether the path exist on `api.get_url()`. I would presume that in practical application user would like to follow up on `get_url` with download or something similar.
@efiop what do you think?

Reply:
@pared `get_url` might not have access to the remote to be able to check for existence. I would rather not do that for now.